backend/Events/VoteEvents.js

```diff
@@ -0,0 +1,57 @@
+const Player = require('../domain/Player');
+const ListVoteDTO = require('../domain/DTO/response/ListVoteDTO');
+const MafiaGame = require('../domain/MafiaGame');
+
+/**
+ * Event handler and logic for `day-vote`
+ * The goal of these vote events is to allow a player to vote for another player and every
+ * other player to receive a list of votes.
+ * @param {any} io
+ * @param {any} socket
+ * @param {MafiaGame} mafiaGame
+ */
+function voteDay(io, socket, mafiaGame) {
+    socket.on('day-vote', (voteForDTO) => {
+        const room = mafiaGame.gameRoomsDict[socket.player.roomID];
+        const voter = socket.player;
+        const votee = voteForDTO.voteFor;
+        room.voteMapping[voter.nickname] = votee;
+        io.in(socket.player.roomID).emit(
+            'day-vote-update',
+            new ListVoteDTO(room.voteMapping)
+        );
+    });
+}
+
+/**
+ * Event handler and logic for `trail-vote`
+ * The goal of these vote events is to allow a player to vote for another player and every
+ * other player to receive a list of votes.
+ * @param {any} io
+ * @param {any} socket
+ * @param {MafiaGame} mafiaGame
+ */
+function voteTrail(io, socket, mafiaGame) {
```

Comment:
rename function to "voteTrial" instead of "voteTrail"

Reply:
Yes, integration will happen when everything is up. Will make required changes.

Reply:
> Logic LGTM aside from a single nitpick with function naming. May be good to integrate into @seif-y 's VoteHandler class in the future.

I reckon we can create a new issue to integrate this with VoteHandler, and then work on it after mine and Justin's stuff has been merged.